src/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ target_link_libraries(hdf5_vfd_gds
   ${HDF5_VFD_GDS_EXPORTED_LIBS}
```

Comment:
```suggestion
  PUBLIC
  ${HDF5_VFD_GDS_EXPORTED_LIBS}
```
Need to label the others in order to use PRIVATE. Assuming these are PUBLIC? Be good if someone could clarify.

Reply:
Yes, those should be public